content/en/docs/concepts/storage/volumes.md

```diff
@@ -370,19 +370,44 @@ spec:
 
 ### gitRepo
 
-A `gitRepo` volume is an example of what can be done as a volume plugin.  It
-mounts an empty directory and clones a git repository into it for your pod to
-use.  In the future, such volumes may be moved to an even more decoupled model,
-rather than extending the Kubernetes API for every such use case.
-
-Here is an example for gitRepo volume:
+While Kubernetes has a built-in `gitRepo` volume type, its functionality is
```

Comment:
Maybe here, the doc should say gitRepo volume type is deprecated.   And then point to a new doc, maybe under tasks?

Reply:
What about adding this example to the init containers doc? https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#what-can-init-containers-be-used-for

Reply:
I think it might be better as a new task doc.  I think users will want to know "how do I access git in my container", and won't think to go to the init containers page.